spec/draft-barnes-cfrg-4q.md

```diff
@@ -350,16 +357,14 @@ and the multiple representations helps save time by avoiding redundant
 computations: the conversion of the first argument to ADD can be done once
 if the argument will be used in multiple additions.
 
-These addition laws are complete: they have no exceptional cases, and
+Below, we list the explicit formulas for the required point
+operations. These formulas, which are adapted from {{Twisted}} and
+{{TwistedRevisited}}, are complete: they have no exceptional cases, and
 therefore can be used in any algorithm for computing scalar multiples
-without worries about wrong answers for rare points. Note that we do not
+without worrying about exceptional procedure attacks {{Exceptional}}. Note that we do not
 explicitly note the point format every time an addition or doubling is used,
 and assume that conversions are done when required.
```

Comment:
We should be explicit about these conversions, since the running state does change representation through the course of the operations.